src/main/scala/com/crealytics/spark/excel/DefaultSource.scala

```diff
@@ -44,6 +45,7 @@ class DefaultSource extends RelationProvider with SchemaRelationProvider with Cr
     data: DataFrame
   ): BaseRelation = {
     val path = checkParameter(parameters, "path")
+    val writeToFile = parameters.get("writeToFile")
```

Comment:
Can we unify `readFromFile` and `writeToFile` to something like `fileLocation` or `filePath`?